src/utilities/indexer/updateAttestationCreated.ts

```diff
@@ -0,0 +1,54 @@
+import { type ICType } from '@kiltprotocol/sdk-js';
+
+import { Op } from 'sequelize';
+
+import { CType as CTypeModel } from '../../models/ctype';
+
+import { logger } from '../logger';
+
+import { matchesGenerator } from './queryFromIndexer';
+
+interface QueriedAttestationCount {
+  cTypeId: ICType['$id'];
+  attestationsCreated: number;
+  registrationBlockId: string; // Block Ordinal Number, without punctuation
+}
+
+export async function updateAttestationsCreated() {
+  // When modifying queries, first try them out on https://indexer.kilt.io/ or https://dev-indexer.kilt.io/
+
+  const fields = ['cTypeId: id', 'attestationsCreated', 'registrationBlockId'];
+
+  const queryParams = {
+    entity: 'cTypes',
+    alias: 'attestationsCreated',
+    fields,
+  };
+
+  const entitiesGenerator =
+    matchesGenerator<QueriedAttestationCount>(queryParams);
+
+  for await (const entity of entitiesGenerator) {
+    const { cTypeId, attestationsCreated } = entity;
+
+    const [affectedCount] = await CTypeModel.update(
+      { attestationsCreated },
+      {
+        where: {
+          id: {
+            [Op.eq]: cTypeId,
+          },
+          attestationsCreated: {
+            [Op.ne]: attestationsCreated,
+          },
+        },
+      },
+    );
+
+    if (affectedCount !== 0) {
```

Comment:
I wonder: if you skip the `ne` query here, does the database still return `affectedCount: 0`?

Reply:
Nope, it is not `0`. It is always 1 (unless it is a new cType). So the `attestationCreated` of all cTypes get overwritten with their previous values.

(I just tried out to be sure.)